<launch>
    <node name="rosbag" pkg="rosbag" type="play"  args="--delay=2 /home/izhleba/hdd/ws/slam-constructor/data/kitti_03.bag"/>
    <env name="ROSCONSOLE_CONFIG_FILE" value="/home/izhleba/hdd/ws/slam-constructor/exp/launch/info.conf"/>

    <node name="vis_odometry_test" pkg="vis_odometry_test" type="odom_listener" output="screen">
        <param name="image" value="/kitti_stereo/left/image_rect"/>
        <param name="info" value="/kitti_stereo/left/camera_info"/>
        <param name="odometry" value="vis_odometry_out"/>
    </node>

    <node name="odometry_estimator" pkg="odometry_estimator" type="odometry_estimator" output="screen">
        <param name="expected" value="/tf"/>
        <param name="expected_tf_frame_id" value="/start"/>
        <param name="actual" value="/vis_odometry_out"/>
        <param name="age_penalty" value="1000"/>
        <param name="out" value="/home/izhleba/hdd/ws/slam-constructor/data/results/kitti_03.csv"/>
     </node>
</launch>
